src/utils/transaction.js

```diff
@@ -285,12 +272,8 @@ export default {
 
             var conId = base58_1.default.decode(contraId)
             var firstArr = [132]
-            var secondArr = []
-            for (var i = 1; i<conId.length-4; i++) {
-                secondArr[i-1] = conId[i]
-            }
+            var secondArr = Array.from(conId.slice(1,conId.length-4))
```

Comment:
.slice() already give you deep copy, no need to put Array.from() outside.

Reply:
If you don't use Array to transfer, there will be something wrong in secondArr.concat() function